### DISCUSSION TITLE
[Feature request] Grid mode (or more than 1 line)

### DISCUSSION THREAD

**Discussion by Triberpunk** [complexlogic/flex-launcher#27] [Ideas]
Id love the ability to add 2 or 3 lines worth of icon/titles in a specific menu
(maybe each menu has either a "line" or "grid" style?)

Not sure if this strays from your vision of the app

Otherwise keep up the great work!

**Comment by complexlogic** [comment]
Thanks for the feedback. There is an open issue #25 for this feature request, so future discussions can occur there.
